src/runtime/schema.ts

```diff
@@ -16,6 +16,7 @@ import {Dictionary} from './hot.js';
 import {CRDTEntity, SingletonEntityModel, CollectionEntityModel} from './crdt/crdt-entity.js';
 import {Referenceable} from './crdt/crdt-collection.js';
 import {CRDTSingleton} from './crdt/crdt-singleton.js';
+import {Flags} from './flags.js';
```

Comment:
schema.ts no longer exists, so you'll have to fix this to merge. Though the debate rages about how to break the schema/type cycle.

Reply:
Thanks, had to take a couple of tries (thanks to sigh not working on my laptop at home) but it's fixed now :+1: